cardano-cli/src/Cardano/CLI/Type/Common.hs

```diff
@@ -489,11 +491,14 @@ data OutputFormatJsonOrText
   | OutputFormatText
   deriving (Eq, Show)
 
-data AllOutputFormats
-  = FormatJson
-  | FormatText
-  | FormatCBOR
-  deriving Show
+data FormatJson = FormatJson
```

Comment:
Maybe it should be an open data family? It could be also a single sum type with `DataKinds`, if we don't need open union.

Reply:
Probably.  We can investigate a single sum type in a future PR.  Seems the change won't be too invasive when we get around to it.